test/action/remove.cpp

```diff
@@ -0,0 +1,57 @@
+// Range v3 library
+//
+//  Copyright Andrey Diduh 2019
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/ericniebler/range-v3
+
+#include <vector>
+#include <range/v3/action/remove.hpp>
+
+#include "../simple_test.hpp"
+#include "../test_utils.hpp"
+
+struct Data{
+    int i;
+
+    Data() = default;
+    explicit Data(int i) : i(i) {}
+
+    Data(const Data&) = delete;
```

Comment:
It seems anomalous to explicitly `delete` one of the copy operations and not the other - it makes me thing something weird is going on here. Could we either explicitly delete both, or allow both to be implicitly deleted (which is the effect of defaulting the move operations)?